frontend/containers/DocumentEdit/DocumentEdit.jsx

```diff
@@ -158,9 +156,7 @@ class DocumentEdit extends Component {
       fields: fields.other
     }]
     const activeSection = this.props.section || sections[0].slug
-    const hasValidationErrors = Object.keys(document.validationErrors).filter(field => {
-      return document.validationErrors[field]
-    }).length
+    const hasValidationErrors = Object.keys(document.validationErrors).filter(field => document.validationErrors[field]).length
```

Comment:
I think it's fine to get rid of the `return` statement, but in that case I'd make it into two lines.